vendor/fbtransform/transforms/xjs.js

```diff
@@ -11,6 +11,51 @@
 var Syntax = require('jstransform').Syntax;
 var utils = require('jstransform/src/utils');
 
+function commaAfterLastParen(value) {
+    var state = "normal";
+    var commaPos = 0;
+    for (var i=0;i<value.length;++i) {
+        if (state === "normal") {
```

Comment:
You have 3 else-branches with the same `commaPos = i+1;`, it should be possible to reduce and flatten this unless there's a very good reason for this verbosity.